src/filters.ts

```diff
@@ -149,7 +149,7 @@ angular.module('3ema.filters', [])
                             cssClass = 'me';
                         } else {
                             const contact = webClientService.contacts.get(possibleMention.substr(2, 8));
-                            if (contact !== null) {
+                            if (contact !== null && contact !== undefined) {
```

Comment:
Oh JS :sob:

Reply:
don't know about typecrypt, but when comparing to `undefined` kit is, AFAIK, usually recommend to compare the [type to the "one of" undefined](https://stackoverflow.com/questions/2778901/how-to-compare-variables-to-undefined-if-i-don-t-know-whether-they-exist#2779104), i.e., add `typeof` and compare it as a string.

Reply:
@rugk That is obsolete with ES2015.

Reply:
Great to hear that! 😄